lib/structures/Message.js

```diff
@@ -272,6 +274,10 @@ class Message extends Base {
         return (this._channelMentions = (this.content.match(/<#[0-9]+>/g) || []).map((mention) => mention.substring(2, mention.length - 1)));
     }
 
+    get jumpLink() {
+        return `${Endpoints.CLIENT_URL}${Endpoints.MESSAGE_LINK(this.channel.type === 1 ? "@me" : this.channel.guild.id, this.channel.id, this.id)}`;
```

Comment:
```suggestion
        return `${Endpoints.CLIENT_URL}${Endpoints.MESSAGE_LINK(this.guildID || "@me", this.channel.id, this.id)}`;
```
Just making it a little more concise

Reply:
Ah, I like this

Reply:
Ok, so, stupid of me to test AFTER committing, but it seems this isn't a valid property. I might have to revert this commit

Reply:
It definitely is valid..